Fireworks/Core/src/FWEventItemsManager.cc

```diff
@@ -212,6 +214,8 @@ FWEventItemsManager::setFrom(const FWConfiguration& iFrom)
 
    if (keyValues == 0) return;
 
+   std::list<FWEventItem*> newItems;
```

Comment:
Is `std::list` appropriate here?  Can we instead use `std::vector`, sized appropriately?